extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/HttpConfiguration.java

```diff
@@ -237,6 +240,17 @@
     @ConfigItem
     public boolean enableDecompression;
 
+    /**
+     * List of media types for which the compression should be enabled automatically, unless declared explicitly via {@link Compressed} or {@link Uncompressed}. 
+     */
+    @ConfigItem(defaultValue = "text/html,text/plain,text/xml,text/css,text/javascript,application/javascript")
+    public Optional<List<String>> compressMediaTypes;
+    
+    /**
+     * The compression level used when compression support is enabled.
+     */
+    public OptionalInt compressionLevel;
```

Comment:
Should we have both enableCompression and compressionLevel? Should we infer "enable" if the level is set?

Reply:
Hm, I don't think it's convenient to say: use `quarkus.http.compression-level=6` to enable compression with the default compression level. In other words, `enable=true` should work in most cases and is simple and expressive.